server/services/curiousService.test.ts

```diff
@@ -69,25 +69,58 @@ describe('curiousService', () => {
       expect(curiousClient.getLearnerNeurodivergence).toHaveBeenCalledWith(prisonNumber, systemToken)
     })
 
-    it('should not prisoner support needs given Curious returns an error', async () => {
+    it('should handle retrieval of prisoner support needs given Curious returns an unexpected error', async () => {
       // Given
       const prisonNumber = 'A1234BC'
       const username = 'a-dps-user'
       const systemToken = 'a-system-token'
       hmppsAuthClient.getSystemClientToken.mockImplementation(() => Promise.resolve(systemToken))
 
-      curiousClient.getLearnerProfile.mockImplementation(() => Promise.reject(Error('Not Found')))
+      const expectedSupportNeeds = {
+        problemRetrievingData: true,
+        healthAndSupportNeeds: undefined,
+        neurodiversities: undefined,
+      } as PrisonerSupportNeeds
+      curiousClient.getLearnerProfile.mockImplementation(() => Promise.reject(Error('Unavailable')))
 
       // When
       const actual = await curiousService.getPrisonerSupportNeeds(prisonNumber, username).catch(error => {
         return error
       })
 
       // Then
-      expect(actual).toEqual(Error('Not Found'))
+      expect(actual).toEqual(expectedSupportNeeds)
+      expect(curiousClient.getLearnerProfile).toHaveBeenCalledWith(prisonNumber, systemToken)
+      expect(curiousClient.getLearnerNeurodivergence).not.toHaveBeenCalled()
     })
   })
 
+  // TODO figure out how to throw a 404 not found error
```

Comment:
Nice - I think we need a little story on the backlog to address 404's from Curious 👍 
I'll add one shortly 👍

Reply:
I've created RR-155